src/Model/Delete/DeletePolicy.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace TractorCow\Fluent\Model\Delete;
+
+use SilverStripe\ORM\DataObject;
+
+interface DeletePolicy
+{
+    /**
+     * @param DataObject $record
+     * @return bool Determines if any dependent objects block upstream deletion (e.g. db / model constraints)
```

Comment:
Is this correct?

Reply:
Yes and it's annoying, but it's the main reason I had to create a nested policy; The use case where an object had both extensions was a pain to solve.